docker/root/custom-cont-init.d/defaults

```diff
@@ -31,6 +31,8 @@ if [ "$CRON_SCHEDULE" != "" ] ; then
 
     # create cron script wrapper
     echo '#!/bin/bash' > "$CRON_WRAPPER_SCRIPT"
+    # Echo commands for easier user debugging:
+    echo "set -x" >> "$CRON_WRAPPER_SCRIPT"
```

Comment:
while this is helpful for us devs, I dont think the user should see this. Let's remove

Reply:
I added that for users to help them debug their cron script. The user's script is outside the image's control but the wrapper is under it's control and there are user errors where echoed commands could help them understand their mistake. Thoughts?